docs/reference/ingest/processors/geoip.asciidoc

```diff
@@ -310,3 +313,36 @@ The `geoip` processor supports the following setting:
     The maximum number of results that should be cached. Defaults to `1000`.
 
 Note that these settings are node settings and apply to all `geoip` processors, i.e. there is one cache for all defined `geoip` processors.
+
+===== Cluster Settings
+
+The `geoip` processor supports the following cluster settings:
+
+`ingest.geoip.downloader.endpoint`::
+
+   (<<static-cluster-setting,Static>>) The endpoint used to update databases used by `geoip` processor. Defaults to https://geoip.elastic.co/v1/database
+
+
+`ingest.geoip.downloader.enabled`::
+
+    Set to `true` (default) to enable automic updates of GeoIP databases. If set to `false` all downloaded databases will be removed.
+
+`ingest.geoip.downloader.poll.interval`::
+
+    How often there will be a check for new databases. Defaults to `3d`, minimum is `1d`
+
+[[ingest-geoip-air-gapped]]
+==== Updating GeoIP databases in air-gapped environments
```

Comment:
Maybe we should both bundle file based updating of databases and using proxy/geoip tool in one section called: 'using custom databases'? Which is for use cases when custom databases are used or Elasticsearch is used in air-gapped environments. This will have 2 sections, one for updating databases via config directory (which replaces the third paragraph of geoip page) and one for setting up a http proxy.

Then the third paragraph in the geoip page can removed, which isn't that important for most folks, since they will be using the Elastic managed databases.